types/ros2d/index.d.ts

```diff
@@ -0,0 +1,400 @@
+// Type definitions for ros2d.js 0.9.0
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).